roles/openshift_openstack/tasks/unprovision.yml

```diff
@@ -1,4 +1,30 @@
 ---
+- name: Get kuryr services
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+  oc_obj:
+    kind: endpoints
+    state: list
+    all_namespaces: true
+  register: svc_output
+  ignore_errors: true
+
+- name: Delete kuryr LoadBalancers (cascading)
+  os_lbaas_deletion:
+    lbaas_annotation: "{{ item.metadata.annotations }}"
+  with_items: "{{ svc_output.results.results[0]['items'] }}"
```

Comment:
Ah so when the previous task  fails, this will fail as well because there will be no `results` in `svc_output`.

And this can't be solved by the `when` block below because that applies to the items being iterated over, not this :-(.

Here's what worked for me:

    with_items: "{{ svc_output.results.results[0]['items'] if 'results' in svc_output else [] }}"

I.e. we use the ternary operator and if there's no results, we return an empty array.

Reply:
Nice catch! thanks!